### PR TITLE
[olm] Remove olm-namespace flag from install sub-command

### PR DESCRIPTION
**Pull request by varshaprasad96**
**Description of the change:**
Remove 'olm-namespace' flag from 'operator-sdk olm install' command.

**Motivation for the change:**
The olm manifests published in github have a hardcoded
namespace value. Hence, the olm operators can only be installed in `olm`
namespace using `operator-sdk olm install` command.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [x] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [x] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

File: internal/cmd/operator-sdk/olm/install.go [hunk 1]
```diff
@@ -35,8 +35,6 @@ func newInstallCmd() *cobra.Command {
 	}
 
 	cmd.Flags().StringVar(&mgr.Version, "version", olm.DefaultVersion, "version of OLM resources to install")
-	cmd.Flags().StringVar(&mgr.OLMNamespace, "olm-namespace", olm.DefaultOLMNamespace,
```
Comment (estroz) [inline internal/cmd/operator-sdk/olm/install.go:38]: `mgr.OLMNamespace` is still used in `InstallVersion`. We should remove this field from the manager's struct and pass the hard-coded default `olm`.

Comment (varshaprasad96) [reply internal/cmd/operator-sdk/olm/install.go:38]: `mgr.OLMNamespace` is still used by status and uninstall commands though

Comment (varshaprasad96) [reply internal/cmd/operator-sdk/olm/install.go:38]: `m.initialize()` would fill the `DefaultOLMNamespace` value in manager's struct. So at least for `install` command, the `mgr.OLMNamespace` would always be `olm` when `InstallVersion` is called.

**Comment by jmrodri** [comment]
Help looks good

```
[jesusr@transam operator-sdk{olm-test}]$ build/operator-sdk olm install --help
Install Operator Lifecycle Manager in your cluster

Usage:
  operator-sdk olm install [flags]

Flags:
  -h, --help               help for install
      --timeout duration   time to wait for the command to complete before failing (default 2m0s)
      --version string     version of OLM resources to install (default "latest")

Global Flags:
      --verbose   Enable verbose logging
```

The flag is gone so trying to use it will print the usage. 
```
[jesusr@transam operator-sdk{olm-test}]$ build/operator-sdk olm install --olm-namespace default
Error: unknown flag: --olm-namespace
Usage:
  operator-sdk olm install [flags]

Flags:
  -h, --help               help for install
      --timeout duration   time to wait for the command to complete before failing (default 2m0s)
      --version string     version of OLM resources to install (default "latest")

Global Flags:
      --verbose   Enable verbose logging

FATA[0000] unknown flag: --olm-namespace                
```

**Pull request merged**